src/main.ts

```diff
@@ -40,10 +39,14 @@ async function main() {
             type: 'string',
             describe: 'The file to translate (leave out for stdin)',
           })
+          .option('language', {
+            type: 'string',
+            describe: 'Language ID to transliterate to',
+          })
           .option('python', {
             alias: 'p',
             boolean: true,
-            description: 'Translate snippets to Python',
+            description: 'Translate snippets to Python. This will override --language option.',
```

Comment:
Let's deprecate this option and give it less priority over `--language`

Reply:
Hint, have `makeVisitor` only look at language and deal with the fallback inside the arrow function. I think that means you'll have to give the `markdown` command the same treatment.

Reply:
Thanks, that makes sense. I'll work on this in a week:)